src/WebJobs.Script/ScriptConstants.cs

```diff
@@ -262,6 +262,7 @@ public static class ScriptConstants
         public static readonly string HttpProxyCorrelationHeader = "x-ms-invocation-id";
         public static readonly string HttpProxyTask = "HttpProxyTask";
         public static readonly string HttpProxyScriptInvocationContext = "HttpProxyScriptInvocationContext";
+        public static readonly string CustomHandlerWorkerRuntime = "custom";
```

Comment:
Other runtimes are defined in `RpcWorkerConstants`. We should have this there as well.